.flowconfig

```diff
@@ -1,4 +1,5 @@
 [ignore]
+.*/node_modules/conventional-changelog-core/.*
```

Comment:
damn types... :rofl: 

I think this is the best approach. And everyone should submit PRs to flow-typed or ... what was the TS... erm, DefinitelyTyped? And then use/install them directly.

edit: heh yea, @DefinitelyTyped and @FlowType

> it would mean we would need to make some changes to the README to tell people that they need to install types when they develop!

:+1: